src/vaadin-grid-pro-edit-column.html

```diff
@@ -253,11 +288,17 @@
         }
       }
 
+      /**
+       * @param {!HTMLElement} editor
+       * @return {string | undefined}
+       * @protected
+       */
       _getEditorValue(editor) {
         const path = this.editorType === 'checkbox' ? 'checked' : this.editorValuePath;
         return Polymer.Path.get(editor, path);
       }
```

Comment:
Doesn't this return a boolean when `editorType === 'checkbox'`? Not sure if it may return even other types but I think at least boolean should also be included in the return type. Or maybe use `unknown`?

Reply:
I think using `unknown` would be a good idea here. Changed.